en/log/flight_review.md

```diff
@@ -241,7 +241,7 @@ The *GPS Uncertainty* plot shows information from the GPS device:
 The GPS Noise & Jamming plot is useful to check for GPS signal interferences and jamming.
 The GPS signal is very weak and thus it can easily be disturbed/jammed by components transmitting (via cable) or radiating in a frequency used by the GPS.
 
-> **Tip** USB 3 is known to be an effective GPS jamming source.
+> **Tip** USB 3 is known to be an effective GPS jamming source. Read more about USB 3 RF Interference at ![USB 3.0* Radio Frequency Interference on 2.4 GHz Devices Intel Paper](https://www.intel.com/content/www/us/en/io/universal-serial-bus/usb3-frequency-interference-paper.html)
```

Comment:
@Zarkopafilis Thanks for this. The format of the link has to remove the ! at the beginning because this indicates to markdown that you are adding an image. I like the link, but probably we don't need a full reference here. So perhaps change to:
```
USB 3 is [known to be](https://www.intel.com/content/www/us/en/io/universal-serial-bus/usb3-frequency-interference-paper.html) an effective GPS jamming source.
```